src/client/app/+about/index.ts

```diff
@@ -1,2 +1,6 @@
+/**
+ * This barrel file provides the export for the lazy loaded
```

Comment:
Yeah, we had discussions about this. Barren is a special kind of a facade but with more specific semantics.